data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/stream/StreamScheduler.java

```diff
@@ -102,6 +122,10 @@ private void processStreamPartition(StreamPartition streamPartition) {
                 runConsumer.whenComplete(completeConsumer(streamPartition));
             }
             numOfWorkers.incrementAndGet();
+            if (numOfWorkers.get() % 10 == 0) {
```

Comment:
Not sure if this log is  helpful.

Reply:
Was thinking of the case where no logs show up for a long time and user does not know what's going on, and this could help that.